log4j-core/src/test/java/org/apache/logging/log4j/core/pattern/MessagePatternConverterTest.java

```diff
@@ -121,7 +121,7 @@ public void testLookup() {
                 .setMessage(msg).build();
         final StringBuilder sb = new StringBuilder();
         converter.format(event, sb);
-        assertEquals("bar", sb.toString(), "Unexpected result");
+        assertEquals("${foo}", sb.toString(), "Unexpected result");
```

Comment:
Given the feature is gone, we can probably remove this test as it’s covered elsewhere

Reply:
I didn't want to remove any test for this release. Just change them to prove the change was made.